.github/workflows/js-tests.yml

```diff
@@ -47,5 +47,18 @@ jobs:
           # Run eslint as a standalone command to generate the test report.
           PRE_COMMIT_NO_CONCURRENCY=true SKIP=eslint pipenv run pre-commit run --show-diff-on-failure --color=always --all-files
           make jslint
+      - name: Validate NOTICES
```

Comment:
I wonder if it's worth adding this as a pre-commit hook so that no one has to worry about it anymore. Then this command will be run automatically when creating the commit.

Reply:
The team discussed this yesterday and decided not to - for now, at least. `make notices` is a bit slow: it takes ~800ms to execute (on my fast Macbook Pro), and we didn't want to pay that cost on every commit. The feeling is that NOTICES will actually only rarely be updated, so having an infrequent CI failure is preferable to slowing every commit down.

Of course, if this becomes a pain, we can add it to the precommit hook!